.github/workflows/CI-lint-format.yml

```diff
@@ -0,0 +1,69 @@
+name: Lint-and-Format
+
+run-name: "Workflow CI/CD Steps: lint and format"
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  lint-format:
+    runs-on: warp-ubuntu-latest-x64-8x
+    name: Lint and format
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Cargo lint
+        uses: ./.github/actions/cmd-in-docker
+        with:
+          command: "cargo clippy --release --all-targets -- --deny warnings 2>&1 | tee linting_output.txt"
+          use_cache: "yes"
+          cache_key: "lint-format"
+          lld_install: "yes"
+          skip_wasm_build: "yes"
```

Comment:
See if this option will be useful to add to zkVerify build as well :wink:

Reply:
Yes, I tried to introduce it here. I'll port it ASAP :smile: